yt/odin/bin/yt_odin_webservice/__main__.py

```diff
@@ -151,6 +167,31 @@ def solomon_duration():
     return jsonify({"sensors": duration_sensors + timed_out_sensors})
 
 
+@app.route("/prom")
```

Comment:
Is it default path for `prometheus` ? I wouldn't use abbreviation here.

Reply:
Default path is `/metrics`, but it can be easily changed in scraper config. Let's use `/prometheus` then? I think it would be not bad to underline the fact this handler is for prometheus